azure-monitor-opentelemetry/setup.py

```diff
@@ -86,8 +86,8 @@
     install_requires=[
         "azure-core-tracing-opentelemetry~=1.0.0b10",
         "azure-monitor-opentelemetry-exporter>=1.0.0b15",
```

Comment:
Should we do ~= for exporter as well, in case we ever do a minor release?

Reply:
Done.